packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/routing.js

```diff
@@ -0,0 +1,58 @@
+import { TRACKING_NAME } from '../constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'octavia-load-balancer.loadbalancer.general-information',
+    {
+      url: '/general-information',
+      views: {
+        loadbalancerView: 'octaviaLoadBalancerOverview',
+      },
+      resolve: {
+        breadcrumb: () => null,
+        flavor: /* @ngInject */ (
+          OctaviaLoadBalancerService,
+          projectId,
+          region,
+          loadbalancer,
+        ) =>
+          OctaviaLoadBalancerService.getFlavor(
+            projectId,
+            region,
+            loadbalancer.flavorId,
+          ),
+        network: /* @ngInject */ (
+          OctaviaLoadBalancerService,
+          projectId,
+          region,
+          loadbalancer,
+        ) =>
+          OctaviaLoadBalancerService.getPrivateNetwork(
+            projectId,
+            region,
+            loadbalancer.vipNetworkId,
+          ),
+        subnet: /* @ngInject */ (
+          OctaviaLoadBalancerService,
+          projectId,
+          region,
+          loadbalancer,
+        ) =>
+          OctaviaLoadBalancerService.getSubnet(
+            projectId,
+            region,
+            loadbalancer.vipNetworkId,
+            loadbalancer.vipSubnetId,
+          ),
+        openEditNameModal: /* @ngInject */ ($state) => () => {
+          $state.go(
+            'octavia-load-balancer.loadbalancer.general-information.edit-name',
+          );
+        },
```

Comment:
```suggestion
        goToEditNameModal: /* @ngInject */ ($state) => () => $state.go(
            'octavia-load-balancer.loadbalancer.general-information.edit-name',
          ),
```
As we're changing state, I think the name should be changed.

Reply:
Not sure about that, indeed we change state, but we display it in a modal (as a child state).
Anyway this name does not seem correct, I think we should either use **openEditNameModal** or **goToEditName**.
I would be interested to know what the team think.